src/generate-gateway-key.ts

```diff
@@ -0,0 +1,74 @@
+import { randomBytes } from 'crypto';
+import { PromptObject } from 'prompts';
+import { OperationsRepository } from './types';
+import { runAndHandleErrors } from './utils/cli';
+import { promptQuestions } from './utils/prompts';
+import { readOperationsRepository } from './utils/read-operations';
+import { writeOperationsRepository } from './utils/write-operations';
+
+const questions = (operationsRepository: OperationsRepository): PromptObject[] => {
+  return [
+    {
+      type: 'autocomplete',
+      name: 'name',
+      message: 'What is the name of the API Integration?',
+      choices: Object.keys(operationsRepository.apis).map((api) => ({ title: api, value: api })),
+    },
+    {
+      type: 'autocomplete',
+      name: 'deployment',
+      message: 'For which deployment do you want to generate the gateway keys for?',
+      choices: (prev) =>
+        Object.keys(operationsRepository.apis[prev].deployments).map((deployment) => ({
+          title: deployment,
+          value: deployment,
+        })),
+    },
+  ];
+};
+
+const main = async () => {
+  const httpGateWayKey = randomBytes(48).toString('hex');
```

Comment:
This can be `httpGatewayKey` as "Gateway" is a single word.

Reply:
github autopilot sigh

Reply:
github autopilot sigh